x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/dataframe/analyses/RegressionTests.java

```diff
@@ -221,17 +221,6 @@ public void testExtractJobIdFromStateDoc() {
         assertThat(Regression.extractJobIdFromStateDoc("noop"), is(nullValue()));
     }
 
-    public void testToXContent_GivenVersionBeforeRandomizeSeedWasIntroduced() throws IOException {
```

Comment:
As with outlier detection, the version here is in the config, not the current node version, so this should stay.